### ISSUE TITLE
Original PC music plays instead of enhanced tracks.

### ISSUE THREAD

**Issue by USBDongleGoblin** [elishacloud/Silent-Hill-2-Enhancements#664]
One area where this is most apparent in the cutscene between Eddie and Laura in the bowling ally. The PC track has no reverb while enhanced does.

Not sure if this extends to cutscene audio, other sounds, or other files in general.

[Desktop 2022.11.02 - 22.02.58.06.mp4](https://user-images.githubusercontent.com/73048023/199641252-89edfaae-33f6-4ad9-9a9a-60907a8d016d.mp4)
[d3d8.log](https://github.com/elishacloud/Silent-Hill-2-Enhancements/files/9925441/d3d8.log)

(PS: No spoilers pls this is my first time playigne. I haven't gotten past this part yet. :( )

**Comment by Polymega** [comment]
I don't believe this background music is using the vanilla PC track, but I can check at a later time. 

If you wear headphones and listen to this playthrough (the YouTube link is timed to play right at the start of this cutscene), you can also hear a static/"hissing" noise as soon as the track starts to play: https://youtu.be/ts1QDpRESM8?t=3987

I believe this noise is part of the track itself.

**Comment by USBDongleGoblin** [comment]
The PS2 clip has the aforementioned reverb effect. This is missing from the vanilla PC version and the clip I posted.

This effect is most apparent on the "popping" that plays through out the song as it adds a bit of a tail to the sound.

I've included the two tracks for comparison if that helps (I had to convert them in order to get around GitHub's file size limit):
[Comparison tracks.zip](https://github.com/elishacloud/Silent-Hill-2-Enhancements/files/9933466/Comparison.tracks.zip)
